lib/private/App/AppStore/Fetcher/AppFetcher.php

```diff
@@ -39,22 +39,21 @@
 use Psr\Log\LoggerInterface;
 
 class AppFetcher extends Fetcher {
-	/** @var CompareVersion */
-	private $compareVersion;
-
 	/** @var IRegistry */
-	protected $registry;
+	protected IRegistry $registry;
 
 	/** @var bool */
 	private $ignoreMaxVersion;
 
-	public function __construct(Factory $appDataFactory,
-								IClientService $clientService,
-								ITimeFactory $timeFactory,
-								IConfig $config,
-								CompareVersion $compareVersion,
-								LoggerInterface $logger,
-								IRegistry $registry) {
+	public function __construct(
+		Factory $appDataFactory,
+		IClientService $clientService,
+		ITimeFactory $timeFactory,
+		IConfig $config,
+		private CompareVersion $compareVersion,
+		LoggerInterface $logger,
+		IRegistry $registry,
```

Comment:
```suggestion
		protected IRegistry $registry,
```
Why is this one not handled the same way?

Reply:
It's addressed in the next commit